### ISSUE TITLE
Next video selection 

### ISSUE THREAD

**Issue by kaybay93** [yuliskov/SmartTubeNext#2577]
So after recent update, if a videos selected on the home screen the following video is the next in the paylist. It used to be a random related video.

**Comment by DiePutinDie** [comment]
Yes, thank you for posting this! I kept wondering why the hell I kept seeing the same videos over and over. Definitely part of a recent update. I prefer the random related videos instead of whatever is next on the page I selected the video from

**Comment by kaybay93** [comment]
Found a potential fix. Go to Settings - Video Player - Misc then uncheck 'Use current selection contents as a playlist'. Then when on a video make sure you have the playlist option as 'play videos continuosly'
